src/translations/ru.json

```diff
@@ -6,6 +6,7 @@
     "charging": "Заряжается",
     "returning home": "Возвращается",
     "segment cleaning": "Уборка зоны/комнаты"
```

Comment:
You forgot to add a `,` on the previous line.

Reply:
> You forgot to add a `,` on the previous line.

Yes! Thank you - I will remember.